app/templates/bestuurseenheid/zitting/index.hbs

```diff
@@ -11,184 +11,158 @@
         <br>
         op
         <span property="besluit:geplandeStart" content={{iso-date this.zitting.geplandeStart}} datatype="xsd:dateTime">{{moment-format this.zitting.geplandeStart 'D MMMM YYYY, HH:mm'}}</span>
-      </h1>
-
-      <ul class="steps steps--timeline">
-        {{#if this.zitting.notulen}}
-          <LinkTo @route="bestuurseenheid.zitting.notulen" @model={{this.zitting.id}} @tagName="li" class="step">
-            <div class="step__icon">
-              {{#if this.notulenPublicationDate}}
-                {{moment-format this.notulenPublicationDate 'D'}}
-                <span class="step__icon__sub">{{moment-format this.notulenPublicationDate 'MMM'}}</span>
-              {{else}}
-                <i class="vi vi-news"></i>
-              {{/if}}
-            </div>
-            <div class="step__wrapper">
-              <div class="step__header">
-                <div class="step__header__titles">
-                  <h2 class="step__title">Notulen</h2>
-                  <span class="u-hidden">
-                    <a href={{href-to "bestuurseenheid.zitting.notulen" this.zitting.id}} property="lblodBesluit:linkToPublication">
-                      <span resource={{this.model.notulen.uri}} typeof="foaf:Document" property="besluit:heeftNotulen"/>
-                    </a>
-                  </span>
-                </div>
-                <div class="step__header__info">
-                  <i class="vi vi-arrow"></i>
-                </div>
+      </AuHeading>
+      <ul class="au-c-steps">
+        <li class="au-c-steps__item">
+          {{#if this.zitting.notulen}}
+          <div class="au-o-box au-c-card au-c-card--flex au-c-card--link">
+            <div class="au-c-card__header">
+              <AuBadge @icon="draft" @skin="brand" />
+              <div>
+                <AuHeading @level="2" @skin="4">Notulen</AuHeading>
+                {{#if this.notulenPublicationDate}}
+                  Gepubliceerd op:
+                  <strong>
+                  {{moment-format this.notulenPublicationDate 'D'}}
+                  {{moment-format this.notulenPublicationDate 'MMM'}}</strong>
```

Comment:
Not 100% sure but I think these 2 can now be combined into 1. (this seems to occur in multiple places)